### PR TITLE
Upgrade to hypernova 2.0.0

### PR DESCRIPTION
**Pull request by frozenfung**
Make a patch for new API in hypernova 2.0.0.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -12,11 +12,14 @@ export const renderReact = (name, component) => hypernova({
...
 
   client() {
-    const { node, data } = load(name);
+    const payloads = load(name);
 
-    if (node) {
-      const element = React.createElement(component, data);
-      ReactDOM.render(element, node);
+    if (payloads) {
+      Array.prototype.map.call(payloads, (payload) => {
```
Comment (ljharb) [inline src/index.js]: Just to confirm here - I think `load` returns an object, which `map` can't iterate over?

Comment (frozenfung) [inline src/index.js]: I doubt `hypernova` change the `load` API in 2.0.0 ?
https://github.com/airbnb/hypernova/blob/master/src/index.js#L81-L88

Comment (ljharb) [inline src/index.js]: Right - that `load` function that you linked returns an object, which can't be iterated over using `Array.prototype.map`

Comment (frozenfung) [inline src/index.js]: Could you explain it in detail? 
Cause I thought `load` just return something like `Array.prototype.map.call(nodes, () => ...)`
Should not the syntax above return an array? 
Or there is something else I missed? :P

Comment (ljharb) [inline src/index.js]: Whoops, you're totally right - i misread :-) in this case I think we can just do `payloads.map` then

Comment (frozenfung) [inline src/index.js]: Yeah! I totally forget that `payloads` is a pure array instead of a nodeLists. Adjustment is made.

Comment (frozenfung) [inline src/index.js]: btw, I prefer `forEach` here. How do you think?

**Comment by frozenfung** [comment]
Rebased.

**Pull request merged**